code/__HELPERS/game.dm

```diff
@@ -485,6 +485,59 @@
 			++i
 	return L
 
+/proc/pollCultists(var/mob/living/Nominee) // Cult Master Poll
+	var/time_passed = world.time
+	set waitfor = FALSE
+	var/list/yes_voters = new
+	var/list/cult_total = new
+	if(world.time<1800)
+		Nominee << "It would be premature to select a leader while everyone is still settling in, try again in [(1800-world.time)/10] seconds"
+		return
+	for(var/datum/mind/B in SSticker.mode.cult)
+		if(!istype(B.current, /mob/living))
+			continue
+		var/mob/living/M = B.current
+		if(isliving(M))
+			M << 'sound/misc/notice2.ogg' //Alerting them to their consideration
+			M.verbs -= /mob/living/proc/cult_master
+			M << "<span class='cultlarge'>[usr] has asserted that they are worthy of leading the cult. A vote will be called shortly.</span>"
+			spawn(200)
+			switch(askuser(M,"[Nominee] seeks to lead your cult, do you support  them?","Please answer in 20 seconds!","Yes","No","Abstain", StealFocus=0, Timeout=200))
+				if(1)
+					M << "<span class='notice'>Choice registered: Yes.</span>"
+					if((world.time-time_passed)>400)
+						M << "<span class='danger'>Sorry, you were too late for the consideration!</span>"
+						M << 'sound/machines/buzz-sigh.ogg'
+					else
+						yes_voters += M
+						cult_total += M
+				if(2)
+					M << "<span class='danger'>Choice registered: No.</span>"
+					cult_total += M
+				if(3)
+					M << "<span class='danger'>Choice registered: Abstain.</span>"
+	sleep(400)
+	if(yes_voters.len > (cult_total.len/2))
+		var/datum/action/innate/cultmast/finalreck/FinalReckoning= new()
+		var/datum/action/innate/cultmast/cultmark/Mark= new()
+		FinalReckoning.Grant(Nominee)
+		Mark.Grant(Nominee)
+		Nominee.update_action_buttons_icon()
+		Nominee.mind.special_role = "Cult Master"
+		SSticker.mode.set_antag_hud(Nominee,"cultmaster")
+		GLOB.cult_mastered = 1
+		for(var/datum/mind/B in SSticker.mode.cult)
+			if(isliving(B.current))
+				B.current << "<span class='cultlarge'>[Nominee] has the cult's support and is now their master. Follow their orders to the best of your ability!"
+		return TRUE
+	else
+		for(var/datum/mind/B in SSticker.mode.cult)
+			if(isliving(B.current))
+				B.current << "<span class='cultlarge'>[Nominee] could not win the cult's support and shall continue to serve as an acolyte. A new vote may be called shortly."
+				sleep(450)

```

Comment:
without a spawn this is going to grant each person their verb back one by one separated by 450 seconds.

just use the add_timer and a callback method that adds the verb